collect_app/src/androidTest/java/org/odk/collect/android/dao/InstancesDaoTest.java

```diff
@@ -195,6 +196,35 @@ public void updateInstanceTest() {
         assertThat(formWithGeopointInstance.getGeometry(), is(nullValue()));
     }
 
+    @Test
+    public void getValuesFromInstanceObjectTest() {
```

Comment:
Is your thinking that this test prevents someone accidentally removing one of the db fields from the method? Is this likely? It seems like it doesn't help prevent the kind of problem we saw here -- a new db field not being represented in the method.

Does it have to be a connected test? Seems it could be Robolectric-based?